docs/testcases/UI/Advanced_Search/1837_8.md

```diff
@@ -0,0 +1,48 @@
+# Search for Tools Group by the attributes.
+
+**Prerequisites**:
+- admin user
+- perform 1837_1, 1838_2 cases
+- Tool `tool1`
+
+**Preparations:**
+
+1. Login as admin user from the prerequisites.
+2. Open **Tools** page.
+3. Open Tool `tool1`
+4. Open **Attributes** panel and click **+Add** button.
+5. Click into the **Key** field and select `Dict1` name from dropdown list.
+6. Click into the **Value** field and select `Value1` from dropdown list. Click **Add** button.
+
+**Actions**:
+
+1. Click on **Search** icon at the left menu bar.
+2. Navigate to `Dict1` filter.
+3. Select `Value1` control.
+4. Hover over the found `tool1` in an appeared list
```

Comment:
Change the action to the actual current one (_click the 'info' icon_)

Reply:
Done